lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -33,10 +37,10 @@
 #include "llvm/Support/raw_ostream.h"
 
 using namespace glow;
+using llvm::StringRef;
```

Comment:
You are trying to get rid of `llvm::StringRef` in the other PR, right?

Reply:
Yup :-).  Will update this one once it lands.